src/config.rs

```diff
@@ -147,7 +155,8 @@ impl From<NoiseSuppressionLevel> for ffi::NoiseSuppression_SuppressionLevel {
 }
 
 /// Noise suppression configuration.
-#[derive(Clone)]
+#[derive(Debug, Copy, Clone, PartialEq)]
```

Comment:
Remove `Copy`?

Reply:
Good catch, I was just starting to place it wherever 😄